### PR TITLE
CRM-20539 Do not heed dao default for field when setting defaults on report

### PR DESCRIPTION
**Pull request by eileenmcnaughton**
* [CRM-20539: Do not heed dao default for field when setting defaults on reports](https://issues.civicrm.org/jira/browse/CRM-20539)

### CODE REVIEW

File: CRM/Report/Form.php [hunk 1]
```diff
@@ -690,16 +690,7 @@ public function preProcess() {
     }
 
     foreach ($this->_columns as $tableName => $table) {
-      // set alias
```
Comment (eileenmcnaughton) [inline CRM/Report/Form.php]: this is just an extraction for readability

File: CRM/Report/Form.php [hunk 2]
```diff
@@ -714,7 +705,7 @@ public function preProcess() {
         }
       }
 
-      $doNotCopy = array('required');
+      $doNotCopy = array('required', 'default');
```
Comment (eileenmcnaughton) [inline CRM/Report/Form.php]: this is the change - ie. if the DAO defaults to 1 then that doesn't mean the field should default to displayed or being a group by

**Comment by seamuslee001** [comment]
Jenkins test this please

**Review by jitendrapurohit — APPROVED** [review]
Tested and agree that the default is not set from the DAO fields, hence not needed while copying.

**Comment by eileenmcnaughton** [comment]
thanks!

**Pull request merged**